### ISSUE TITLE
Opt-in for error logging (talkback service)

### ISSUE THREAD

**Issue by ktomala** [golemfactory/golem-electron#244]
Since we have talkback service setup currently, we need to add an error logging opt-in mechanics inside golem-electron. On the backend side the issue https://github.com/golemfactory/golem/issues/2298 was created. 

What we need in golem-electron is:
* checkbox on application startup for opt-in
* option in settings to show/change opt-in status
* trigger on change setting that will issue RPC request to golemapp
* store configuration parameter for Sentry DSN inside golem-electron
* add logging configuration in NodeJS (https://docs.sentry.io/clients/javascript/integrations/electron/)
* disable/enable talkback logging in NodeJS based on opt-in parameter status
* possibly add some information about what we log

**Comment by maaktweluit** [comment]
We could use the `--nomonitor` `golemapp` argument for this, currently by default data is send to the monitor.

**Comment by ktomala** [comment]
This is something else, I believe we should have a distinctive option. And since this is opt-in (not opt-out) we would prefer rather --enable-talkback or something like this.

**Comment by maaktweluit** [comment]
I think all data collection should be turned off with an opt-in, we can make them separate arguments / settings but both should be a users choice.

**Comment by ktomala** [comment]
Just to clarify: Opt-in = turn on data collection.

**Comment by ktomala** [comment]
We have opt-in implemented in golemapp via https://github.com/golemfactory/golem/pull/2375.

Updated via rpc with `settings set enable_talkback {0,1}`
